src/openagi/llms/groq.py

```diff
@@ -1,14 +1,11 @@
 from typing import Any
-from langchain_core.messages import HumanMessage
+from groq import Groq
```

Comment:
import groq under the exception block. or you can also import it under ``load`` function. 

Refer: [https://github.com/aiplanethub/beyondllm/blob/main/src/beyondllm/llms/chatgroq.py](https://github.com/aiplanethub/beyondllm/blob/main/src/beyondllm/llms/chatgroq.py)